### PR TITLE
NavigationBar and styling

### PR DESCRIPTION
**Pull request by Pradyumna369**

### CODE REVIEW

File: src/Main.tsx [hunk 1]
```diff
@@ -1,9 +1,15 @@
 import React from "react";
+import NavigationBar from "./components/NavigationBar";
+import HomePage from "./pages/HomePage";
 
 
 function Main() {
     return (
-        <>Hello World!</>
+        <>
```
Comment (sreekar-ss) [inline src/Main.tsx]: `npm run prettier:write`

Comment (sreekar-ss) [reply src/Main.tsx]: cleans up code and presents in a readable format

File: src/components/NavigationBar.tsx [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+import PersonOutlineOutlinedIcon from '@mui/icons-material/PersonOutlineOutlined';
+import "../css/HomePage.css"
+
+
+export default function SimpleBottomNavigation() {
+  const [value, setValue] = React.useState(0);
+
+  return (
+    <div>
+    <div className='header'>
```
Comment (sreekar-ss) [inline src/components/NavigationBar.tsx]: remove this line

File: src/components/NavigationBar.tsx [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    </div>
+    
+    <div className='NavigationBar'>
+      <BottomNavigation
+        showLabels
+        value={value}
+        onChange={(event, newValue) => {
+          setValue(newValue);
+        }}
+        className='BottomNavigation'
```
Comment (sreekar-ss) [inline src/components/NavigationBar.tsx]: use camelCase always `bottomNavBar`

File: src/components/NavigationBar.tsx [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+import AddCircleIcon from '@mui/icons-material/AddCircle';
+import GroupsRoundedIcon from '@mui/icons-material/Groups';
+import HomeRoundedIcon from '@mui/icons-material/HomeRounded';
+import NotificationsOutlinedIcon from '@mui/icons-material/NotificationsOutlined';
+import PersonOutlineOutlinedIcon from '@mui/icons-material/PersonOutlineOutlined';
+import "../css/HomePage.css"
+
+
+export default function SimpleBottomNavigation() {
+  const [value, setValue] = React.useState(0);
```
Comment (sreekar-ss) [inline src/components/NavigationBar.tsx:12]: remove unused `value`

File: src/Main.tsx (hunk 1 above)
Comment (sreekar-ss) [inline src/Main.tsx]: use `<div></div>` for the parent block

File: src/css/HomePage.css [hunk 5]
```diff
@@ -0,0 +1,38 @@
+div .NavigationBar{
+    width: 100%;
+    display: block;
+}
+
+div .header{
+    background-color: black;
+    height: 30px;
```
Comment (sreekar-ss) [inline src/css/HomePage.css]: use `em` instead of `px`

File: src/css/HomePage.css [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+}
+
+div .NavigationBar .BottomNavigation{
+    background-color: black;
+}
+
+@media(max-width:700px) {
```
Comment (sreekar-ss) [inline src/css/HomePage.css]: make it `500px`

Comment (sreekar-ss) [reply src/css/HomePage.css]: @media screen (max-width: 500px) {}

File: src/pages/HomePage.tsx [hunk 7]
```diff
@@ -1,11 +1,12 @@
 import React from "react";
-
+import "../css/HomePage.css"
```
Comment (sreekar-ss) [inline src/pages/HomePage.tsx]: this should be imported inside `App.css`

File: src/components/NavigationBar.tsx [hunk 8]
```diff
@@ -0,0 +1,40 @@
...
+        className='BottomNavigation'
+      >
+        <BottomNavigationAction label="Home" icon={<HomeRoundedIcon/>} sx={{ color: 'grey' }} />
+        <BottomNavigationAction label="Friends" icon={<PersonOutlineOutlinedIcon/>} sx={{ color: 'grey' }}/>
+        <BottomNavigationAction label="Add" icon={<AddCircleIcon className='Add'sx={{ fontSize: 60 }}/>} sx={{ color: 'grey',height: 20 }}/>
+        <BottomNavigationAction label="Groups" icon={<GroupsRoundedIcon/>} sx={{ color: 'grey' }}/>
+        <BottomNavigationAction label="Updates" icon={<NotificationsOutlinedIcon/>} sx={{ color: 'grey' }}/>
+      </BottomNavigation>
+    </div>
+    <div className='backgroundColor'>
```
Comment (sreekar-ss) [inline src/components/NavigationBar.tsx]: remove

File: src/components/NavigationBar.tsx (hunk 4 above)
Comment (Pradyumna369) [reply src/components/NavigationBar.tsx:12]: Need this value to select the icon which is active and display its css properties

**Pull request closed without merging**